plugins/ResourceFetcherPlugin/main.js

```diff
@@ -0,0 +1,103 @@
+
+// This is a sample implementation of a resource fetcher plugin.  It is intended to show how a resource fetcher plugin might be written.
+// This plugin uses an ajax call to call to a service to get the contents of a resource.
+//
+// It's based on a plugin used at Bibliotheca to get the contents of an epub from a server.  I've edited this file to remove
+// some functionality specific to Bibliotheca.  The result may or may not be suitiable for your needs.
+
+define(['readium_js_plugins', 'readium_shared_js/globals', '../../../js/epub-fetch/discover_content_type' ],
```

Comment:
PS: the point I am making here is that Readium "plugins" were originally designed (minimally) for usage within the `readium-shared-js` scope. So some architectural hacks are required to implement plugins for `readium-js`, etc.
@JCCR